docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/03_instrument_install.md

```diff
@@ -0,0 +1,72 @@
+
+The intermediary state is necessary for the generation of the SQL `CREATE TABLE` patch which, in turn, is necessary for the install process when using dedicated SQL table storage. In order to get from the intermediary serialized format to an SQL table, the `generate_tables_sql.php` tool is then used on the `ip_output.txt` file.
+
+Follow the steps below to install a new PHP instrument.
+
+  1. Move your newly created instrument `NDB_BVL_Instrument_TEST_NAME.class.inc` into the `project/instruments/` directory.
+  2. Navigate to the LORIS `tools/` directory.
+  3. Run the instrument through the `lorisform_parser.php` to parse the PHP instrument into the intermediary serialed format.
+  		```
+  		ls ../project/instruments/NDB_BVL_Instrument_TEST_NAME.class.inc | php lorisform_parser.php
+  		```
+  4. Run the `generate_tables_sql.php` tool to converted the serialized format into an SQL `CREATE TABLE` SQL patch. The patch will be created in the `project/` directory by default.
+  		```
+  		php generate_tables_sql.php
+  		```
+  5. Source the newly generated SQL patch `project/TEST_NAME.sql` into your SQL database to create the table.
+  6. Register the instrument in the `test_names` table of the database by running the following command. Make sure to replace the `'TEST_NAME'` value with the actual test name used in your instrument. 
+  
+  	```
+  	INSERT INTO test_names (Test_name,Sub_group) SELECT 'TEST_NAME', ID FROM test_subgroups WHERE Subgroup_name='Instruments';
+  	```
+  	> _**Note:** The `Sub_group` entry in the query above is necessary for the instrument registration. See the [Instrument Grouping](#instrument-grouping) section below for additional details._
+  		
+By completing these steps the instrument should be installed and ready to be assigned to a battery; only instruments assigned to a behavioural test battery are available for data entry For more information about test batteries, navigate to the [Test Battery](./07_clinical_test_battery.md) section directly or continue following the documentation.
+
+> _**Note:** Each PHP tool mentioned above contains documentation at the top of the file in case additional details are required._
+
+
+### OPTION 2: JSON Data Storage
+**PHP and LINST instruments using JSON data storage** do not require a dedicated SQL table on the database thus making their installation much simpler. Once your instrument is finalized, save it and move it to the `project/instruments/` directory (this is the same location the Instrument Manager stores installed instruments). Once the instrument is in the correct location, you must add it manually to the `test_names` table of the database to register it. The following query can be used as a template for the registration.
```

Comment:
if this is in beta, add that simple caveat (not sure)

Reply:
It's the recommended path at this stage so not beta